src/app/workflow/permissions/permissions.component.ts

```diff
@@ -76,6 +78,7 @@ export class PermissionsComponent implements OnInit {
         .addWorkflowPermission(this.workflow.full_workflow_path, { email: value, role: permission }, this.entryType === EntryType.Service)
         .subscribe(
           (userPermissions: Permission[]) => {
+            this.alertService.detailedSuccess('Permissions updated');
```

Comment:
out of a silly desire for consistency, adding permissions results in "Permissions updated" but removing permissions actually says that permissions were removed :wink:

Reply:
I like consistency. :)